backend/app/dependencies.py

```
@@ -0,0 +1,45 @@
+                status_code=status.HTTP_401_UNAUTHORIZED,
+                detail="User not found",
+                headers={"WWW-Authenticate": "Bearer"},
+            )
+        
+        # Convert ObjectId to string
+        user["_id"] = str(user["_id"])
+        return user
+        
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)